static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-immersive.scss

```diff
@@ -12,6 +12,13 @@
         z-index: 1;
     }
 
+    //Todo better way to get this immersive card type to set the height when sublinks. This is fixed height enough to fit two lines of sublink
```

Comment:
As soon as you throw a `position: absolute` in the game, your layout is f***d. The whole card layout needs to be rethought using features available in modern browsers; will be much more intuitive then.